build.gradle

```diff
@@ -44,18 +44,19 @@ repositories {
 
 dependencies {
 	compile('org.springframework.boot:spring-boot-starter-data-jpa')
-	compile 'mysql:mysql-connector-java'
+	compile('mysql:mysql-connector-java')
 	compile('org.springframework.boot:spring-boot-starter-web') {
 		exclude module: 'spring-boot-starter-tomcat'
 	}
-	compile 'org.springframework.boot:spring-boot-starter-undertow'
+	compile('org.springframework.boot:spring-boot-starter-undertow')
 	compile('com.fasterxml.jackson.module:jackson-module-kotlin')
 	compile("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
 	compile("org.jetbrains.kotlin:kotlin-reflect")
 	compile("com.graphql-java:graphql-spring-boot-starter:4.4.0")
 	compile("com.graphql-java:graphql-java-tools:5.2.0")
 	compile("com.graphql-java:graphiql-spring-boot-starter:4.4.0")
 	runtime('com.h2database:h2')
+	compile('org.springframework.boot:spring-boot-devtools')
```

Comment:
Including this enables auto-restart for spring boot apps, plus some other developer features (https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-devtools.html)